### PR TITLE
fix(driver-adapter-utils): fix `pg has no default export`

### PR DESCRIPTION
**Pull request by jkomyno**
Prompted by: https://github.com/prisma/prisma/issues/23091.

Note: https://github.com/prisma/prisma/issues/23091 is already fixed, we just need to inform the user.

/integration

### CODE REVIEW

File: packages/driver-adapter-utils/src/debug.ts [hunk 1]
```diff
@@ -1,4 +0,0 @@
-import { Debug } from '@prisma/debug'
-
-export { Debug }
-export default Debug
```
Comment (millsp) [inline packages/driver-adapter-utils/src/debug.ts:4]: Wouldn't removing the default export constitute a slight breaking change? Are there driver adapters out there that use it maybe? If not, I think the change is ok.

**Comment by jkomyno** [comment]
Note: the error this PR's issue refers to is already fixed in `5.10.2` and in dev (`5.11.0-dev.20`).
However, we have an other problem now:


```
❯ pnpm tsc ./src/index.ts
node_modules/.pnpm/@prisma+adapter-pg@5.11.0-dev.20_pg@8.11.3/node_modules/@prisma/adapter-pg/dist/index.d.ts:3:8 - error TS1192: Module '"/Users/jkomyno/work/prisma/reprod-9289/node_modules/.pnpm/@types+pg@8.11.2/node_modules/@types/pg/index"' has no default export.

3 import pg from 'pg';
         ~~


Found 1 error in node_modules/.pnpm/@prisma+adapter-pg@5.11.0-dev.20_pg@8.11.3/node_modules/@prisma/adapter-pg/dist/index.d.ts:3
```

Here are the relevant files for reproduction:

```jsonc
// tsconfig.json
{
  "compilerOptions": {
    "rootDir": "./src",
    "outDir": "./dist",
    "allowJs": true,
    "strict": true,
    "noImplicitAny": true,
    "strictNullChecks": true,
    "strictFunctionTypes": true,
    "strictPropertyInitialization": true,
    "strictBindCallApply": true,
    "noImplicitThis": true,
    "noImplicitReturns": true,
    "alwaysStrict": true,
    "esModuleInterop": true,
    "declaration": true,
    "target": "ES6",
    "jsx": "react",
    "module": "commonjs",
    "moduleResolution": "node",
  },
  "include": ["src/**/*.ts"],
  "exclude": ["node_modules"],
  "ts-node": {
    "esm": true, // «———— enabling ESM for ts-node
  },
}
```

```typescript
// ./src/index.ts
import {Pool} from 'pg'
import {PrismaPg} from '@prisma/adapter-pg'
import {PrismaClient} from '@prisma/client'

const connectionString = `${process.env.DATABASE_URL}`

const pool = new Pool({connectionString})
const adapter = new PrismaPg(pool)
const prisma = new PrismaClient({
    log: ['query'],
    adapter,
})
```

```jsonc
// package.json
{
  "name": "reprod-23091",
  "version": "1.0.0",
  "description": "",
  "scripts": {
    "build": "tsc"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@types/node": "^20.11.24",
    "@types/pg": "^8.11.2",
    "prisma": "5.11.0-dev.20",
    "tsx": "^4.7.1",
    "typescript": "^5.3.3"
  },
  "dependencies": {
    "@prisma/adapter-pg": "5.11.0-dev.20",
    "@prisma/client": "5.11.0-dev.20",
    "pg": "^8.11.3"
  }
}
```

**Comment by jkomyno** [comment]
I confirm that `5.11.0-integration-fix-debug-reexport.2` fixes all known compilation issues related to `@prisma/adapter-pg`.

**Comment by jkomyno** [comment]
> I confirm that `5.11.0-integration-fix-debug-reexport.2` fixes all known compilation issues related to `@prisma/adapter-pg`.

The change is simple and fixes the TypeScript compilation issue, I'll merge it.

File: packages/driver-adapter-utils/src/debug.ts (hunk 1 above)
Comment (jkomyno) [reply packages/driver-adapter-utils/src/debug.ts]: Code change reverted.

**Pull request merged**